### PR TITLE
Media react component

### PR DESCRIPTION
**Pull request by matikrk**
* rename avatar component from avatar.jsx to Avatar.jsx
* fix avatar ico size
* media component examples
* delete unused properties of media component
* create media component
* tests

### CODE REVIEW

File: src/components/avatar/Avatar.jsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ const sizes = {
 const iconSizes = {
   xsmall: 14,
```
Comment (vergilius) [inline src/components/avatar/Avatar.jsx]: I still think, that these values should be camelized

Comment (matikrk) [inline src/components/avatar/Avatar.jsx]: for now it's same as style names
maybe later we standardise it in all components, same plan is for colors

File: src/components/media/_media.scss [hunk 2]
```diff
@@ -60,11 +60,6 @@ $mediaItemSecondaryColor: $grayPrimary;
     min-width: 0;
     position: relative;
     flex: 1 1 0;
-
-    &--centered {
-      display: flex;
-      align-items: center;
-    }
```
Comment (aju) [inline src/components/media/_media.scss]: not used?

Comment (matikrk) [inline src/components/media/_media.scss]: not used
https://github.com/search?utf8=%E2%9C%93&q=org%3Abrainly+sg-media__wrapper--centered&type=Code
and also I tried search it manually

**Pull request merged**